backend/Restaurant-System.DataAcessLayer/Models/Table.cs

```diff
@@ -3,21 +3,31 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
 
 namespace Restaurant_System.DataAcessLayer.Models
 {
+
     public class Table
     {
-        public int Id { get; set; }
+        [Key]
+        public int IdTable { get; set; }
+
+        public int IdRestaurant { get; set; }
+
+        public int IdRoom { get; set; }
```

Comment:
Tutaj podobnie dopiero jak zrobimy bazę Room to wtedy możemy zapewnić relację między obiektami. 
Dodatkowo protip: czytelniej jest id dawać na końcu np.RestaurantId itd.. :)